app/models/katello/repository.rb

```diff
@@ -134,7 +136,11 @@ class Repository < Katello::Model
     scope :order_by_root, ->(attr) { joins(:root).order("#{Katello::RootRepository.table_name}.#{attr}") }
     scope :with_content, ->(content) { joins(Katello::RepositoryTypeManager.find_content_type(content).model_class.repository_association_class.name.demodulize.underscore.pluralize.to_sym).distinct }
     scope :by_rpm_count, -> { left_joins(:repository_rpms).group(:id).order("count(katello_repository_rpms.id) ASC") } # smallest count first
-
+    scope :exportable, -> { with_type(EXPORTABLE_TYPES) }
+    scope :immediate_or_none, -> do
+      immediate.or(where("#{RootRepository.table_name}.download_policy" => nil)).
+        or(where("#{RootRepository.table_name}.download_policy" => ""))
```

Comment:
Shouldn't we have a validation that makes it so we don't have to check both `nil` and `""`?

Reply:
well file types have `""` and ansible I think has nil for download policy. I could add a rule that sets that column to nil if it  is ''. But would then have to create a migration for that. I didnt think this merited a migration but I am not averse to it.